cmd/edgex-ui-server/static/js/pages/appService.js

```diff
@@ -96,8 +119,8 @@ orgEdgexFoundry.appService = (function () {
         $(".helper-dialog-wrapper").bind("mouseup",function() {
                 if(leftContainer.find("div[id='"+moveDivId+"']").length == 0){
                     leftContainer.append($("#"+moveDivId));
-                    if($("#"+moveDivId).find("input[type='text']").length == 0){
-                        var button='<button type="button" onclick="clickParamButton()" class="btn btn-success paramButton" value="" id="'+moveDivId+'input" title="'+$("#"+moveDivId)[0].getAttribute("title")+'" placeholder="Set Params" onmouseup="event.cancelBubble = true" onmousedown="event.cancelBubble = true">' +
+                    if($("#"+moveDivId).find("button[type='button']").length == 0){
```

Comment:
please set the variable as prefixed by the property value in edgexfoundry in next time